### DISCUSSION TITLE
Frustrating Reload + Scroll to Top after adding Repo to List.

### DISCUSSION THREAD

**Discussion by michael-azogu** [community/community#20353] [Other Feature Feedback, Questions, & Ideas]
on mobile. anytime I'm in the starred repos view & add a repo to a list, the view reloads & scrolls back to the top.

something to do with virtualizing & memory wipes

**Comment by michael-azogu** [comment]


**Reply by EDIflyer** [reply]
Agree, noticed the same issue here, am using the latest version of the app on Android (v1.71.1 [666])

**Reply by AlyaAlesya** [reply]
Hai

**Reply by Bladfool** [reply]
> > anytime I'm in the starred repos view & add a repo to a list, the view reloads & scrolls back to the top.
> 
> this view specifically
> 
> ![Screenshot_20220712-001328_GitHub.png](https://user-images.githubusercontent.com/73001497/178374618-6b55a293-c281-46d7-8610-55897a3e326b.png)

**Reply by Bladfool** [reply]
> > > anytime I'm in the starred repos view & add a repo to a list, the view reloads & scrolls back to the top.
> > 
> > this view specifically
> > 
> > ![Screenshot_20220712-001328_GitHub.png](https://user-images.githubusercontent.com/73001497/178374618-6b55a293-c281-46d7-8610-55897a3e326b.png)
> 
>

**Comment by michael-azogu** [comment]


**Reply by stefancruz** [reply]
I have the same issue. Came here to report it.

**Comment by hudsontrius80A** [comment]
> on mobile. anytime I'm in the starred repos view & add a repo to a list, the view reloads & scrolls back to the top.

**Comment by AlyaAlesya** [comment]
@keisaacson

**Comment by EnochianD** [comment]
https://github.com/DexterHuang/CyberCodeOnline/tree/master/contribution%2Fitem%2Fcommunity-lore

**Reply by EnochianD** [reply]
![GIF_20220927_053854.gif](https://user-images.githubusercontent.com/110375788/192498466-76e4a743-a015-487b-a03f-d214ddcb4844.gif)

**Comment by Melaniejmcclure94** [comment]
Need my financial picture please

**Comment by Danielsoas** [comment]
have 800+ starred repos & it's a pain to scroll down to where I was & continue that every time I add to a list

**Reply by Danielsoas** [reply]
have 800+ starred repos & it's a pain to scroll down to where I was & continue that every time I add to a list

**Comment by gutwrinch** [comment]
Wed3

**Comment by Mroczek1992** [comment]
Jo

**Comment by jujashr** [comment]
noonaghnjww@michael-azogu

**Reply by jujashr** [reply]
Telegram

**Comment by jujashr** [comment]
- [x] @michael-azogu **[]()**

**Comment by YVS37** [comment]
J

**Comment by Binary-Ninja-007** [comment]
Good.
